code/game/atoms.dm

```diff
@@ -579,6 +579,10 @@
 /atom/proc/get_examine_string(mob/user, thats = FALSE)
 	return "[icon2html(src, user)] [thats? "That's ":""][get_examine_name(user)]"
 
+///Used to insert text after the name but before the discription in examine()
+/atom/proc/get_name_chaser(mob/user, text)
+	return text
```

Comment:
```suggestion
/atom/proc/get_name_chaser(mob/user, list/name_chaser = list())
	return name_chaser.Join()
```
And change the other instances.
Use lists to collect info and `Join()` at the end.
Text appending is awfully inefficient in DM because strings are immutable entities.

Reply:
I agree with typecasting the list, but while my poor var names may not make this clear, text in this case is a list, get_name_chaser is literally just inserting new things into the examine list after the name is set